src/k8s-engine/images/imageProcessor.ts

```diff
@@ -305,13 +305,25 @@ export abstract class ImageProcessor extends EventEmitter {
           if (this.lastErrorMessage !== timeLessMessage) {
             this.lastErrorMessage = timeLessMessage;
             this.sameErrorMessageCount = 1;
-            console.log(result.stderr.replace(/(?!<\r)\n/g, '\r\n'));
+            if (args) {
+              console.log(`> ${ this.processorName } ${ subcommandName } ${ args.join(' ') }:\r\n${ result.stderr.replace(/(?!<\r)\n/g, '\r\n') }`);
+            } else {
+              console.log(`> ${ this.processorName } ${ subcommandName }:\r\n${ result.stderr.replace(/(?!<\r)\n/g, '\r\n') }`);
+            }
```

Comment:
Additionally: should these be `console.debug` instead of `console.log`? (There's a checkbox in the UI to enable logging from `console.debug` lines; they're normally not needed if things are going well.)

But then, given the existing code, I suppose it's `console.log`ing everywhere anyway, so perhaps this is fine.

Reply:
Thanks!  I've added the suggested code change.  As far as console.debug vs console.log, I think console.log works here since Jan commented that he wanted to see all lima commands regardless of debug settings.  I'll definitely need to watch for the difference in the other files.

Reply:
> Jan commented that he wanted to see all lima commands regardless of debug settings.

Yes, thank you! I don't want to have to go back to users to ask them to reproduce an issue once more after enabling debug mode. Debug mode should be reserved for activities that would create too much output that would make the logs unwieldy. The number of calls to `limactl` is relatively small, so we should log all of them unconditionally.